Utils.ps1

```diff
@@ -22,6 +22,9 @@ Set-Alias ?? Invoke-NullCoalescing -Force
 function Invoke-Utf8ConsoleCommand([ScriptBlock]$cmd) {
     $currentEncoding = [Console]::OutputEncoding
     try {
+        # A native executable that writes to stderr AND has its stderr redirected will generate non-terminating
+        # error records if the user has set $ErrorActionPreference to Stop. Override that value in this scope.
+        $ErrorActionPreference = 'Continue'

```

Comment:
> Even if we redirect the error stream to `$null`, it still gets captures in `$Error` e.g.:

And there's no way to suppress that? I'm not thrilled about polluting `$Error` with every `prompt` execution when not in a Git directory. Almost tempted to add code to `Invoke-Utf8ConsoleCommand` to clear out any new `NativeCommandError` entries in `$Error`.